addons/udes_stock/models/stock_move.py

```diff
@@ -384,7 +388,12 @@ def refactor_action_group_by_move_key(self):
                       '') % (key, move_group.mapped('location_id'),
                              move_group.mapped('location_dest_id')))
 
-            Picking._new_picking_for_group(key, move_group)
+            group_pickings = move_group.mapped('picking_id')
+            origin = None
+            if len(group_pickings) == 1:
```

Comment:
This should set the origin if there are multiple group picks with the same origin. Something like:
```
origins = set(group_pickings.mapped('origin'))
origin = None
if len(origins) == 1:
    origin = origins[0]

```